spec/models/site_status_spec.rb

```diff
@@ -168,4 +170,35 @@
       end
     end
   end
+
+  describe "description" do
+    subject { build(:site_status, :running, :unpublished, site: create(:site, location_name: 'Foo', code: '1')) }
+    its(:description) { should eq 'Foo (code: 1) – running/unpublished' }
+  end
+
+  describe "default_vac_status_given" do
+    subject { SiteStatus }
+    it "should return correct default_vac_status" do
+      expect(subject.default_vac_status_given(study_mode: 'full_time')).to eq :full_time_vacancies
+      expect(subject.default_vac_status_given(study_mode: 'part_time')).to eq :part_time_vacancies
+      expect(subject.default_vac_status_given(study_mode: 'full_time_or_part_time')).to eq :both_full_time_and_part_time_vacancies
+      expect { subject.default_vac_status_given(study_mode: 'foo') }.to raise_error("Unexpected study mode foo")
+    end
+  end
+
+  describe "status changes" do
+    describe "when suspending a running, published site status" do
+      subject { create(:site_status, :running, :published).tap(&:suspend!) }
+      its(:status) { should eq("suspended") }
```

Comment:
Just a suggestion, but this would read a little better and also hide the implementation detail:

```
it { should be_status_suspended }
```

Reply:
fixed